koala/ast/graph.py

```diff
@@ -413,13 +453,14 @@ def __init__(self,value,precedence,associativity):
         self.precedence = precedence
         self.associativity = associativity
 
-def shunting_yard(expression, named_ranges):
+def shunting_yard(expression, named_ranges, ref = ''):
```

Comment:
Can you please let a brief comment on shunting_yard method to explain what is 'ref' and it purpose.
This code is actually shared by many open-source contributors and I guess it would be usefull to explain why this specific adjustement is so important